### PR TITLE
Add PR and Issue Title in Notification DMs

### LINKED ISSUE #133: Add PR Title in Notifications DMs
The [Github plugin](https://github.com/mattermost/mattermost-plugin-github) sends DMs for Github notifications related to PRs, i.e. when your PR is approved or when you are requested a review on a PR. The current format of the DM is:

Screen Shot 2019-09-27 at 11 16 24 AM

These type of DMs should be improve to show the PR title as the text of the link to the PR. You can check out the text templates defined for the plugin’s webhooks [here](https://github.com/mattermost/mattermost-plugin-github/blob/master/server/template.go)

----
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://community.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

JIRA: https://mattermost.atlassian.net/browse/MM-18878

### PR DESCRIPTION
**Pull request by iwataka**
fixes #133

### CODE REVIEW

File: server/template.go [hunk 1]
```diff
@@ -153,17 +153,17 @@ func init() {
 `))
 
 	template.Must(masterTemplate.New("commentMentionNotification").Funcs(funcMap).Parse(`
-{{template "user" .GetSender}} mentioned you on [{{.GetRepo.GetFullName}}#{{.Issue.GetNumber}}]({{.GetComment.GetHTMLURL}}):
+{{template "user" .GetSender}} mentioned you on [#{{.Issue.GetNumber}} {{.Issue.GetTitle}}]({{.GetComment.GetHTMLURL}}):
```
Comment (mickmister) [inline server/template.go]: @iwataka Do you think it makes sense to keep the repo name in the message? I think it provides value so the dev knows which repo the PR/issue is in upon inspection.

Comment (marianunez) [reply server/template.go]: I agree that the repo name is useful. Should we do `Name` instead of `Full Name` so it does not get too wordy with the title? 

Something like: 

[mattermost-plugin-github#136](https://github.com/mattermost/mattermost-plugin-github/pull/136) - Add PR and Issue Title in Notification DMs

Thoughts @mickmister

Comment (mickmister) [reply server/template.go]: @marianunez Sounds good to me. I'm trying to think of non-MM org users, and if they frequently have interactions with repos from different orgs. Looking at my DM feed, it certainly seems redundant to have the org show on each message.

:+1: On just including repo name without the org

Comment (iwataka) [reply server/template.go]: @mickmister Thanks for your suggestion and it makes sense.
So I will change template strings like the below:
> mattermost-plugin-github#136 - Add PR and Issue Title in Notification DMs

Comment (iwataka) [reply server/template.go]: Finished @mickmister

Comment (mickmister) [reply server/template.go]: @iwataka FYI, our convention for PRs is to merge instead of rebase, so we can easily tell the differences from the last time we reviewed. No worries though, just a heads up

**Comment by mickmister** [comment]
Thanks @iwataka, great work!

**Comment by prapti** [comment]
@prapti test message. Please ignore.
